app/routes.py

```diff
@@ -71,6 +73,33 @@ def create_kml():
     )
 
 
+@app.route(f'/{ROUTE_ADMIN_PREFIX}', methods=['GET'])
+def get_kml_metadata_by_admin_id():
+    admin_id = request.args.get('admin_id')
```

Comment:
Sorry, too late as it is merged already.
Anyways, I have a discussion on my mind, where it was suggested, to keep `admin_id` out of the URL if possible. Now it seems to be a URL query parameter. So it is not possible to hava a POST endpoint for this instead?